packages/fakta/tilrettelegging/src/components/arbeidsforhold/tilretteleggingOgOpphold/TilretteleggingOgOppholdPerioderPanel.tsx

```diff
@@ -113,15 +122,14 @@ export const TilretteleggingOgOppholdPerioderPanel = ({
                 ? dayjs(nesteTilrettelegging.fom).subtract(1, 'day').format(ISO_DATE_FORMAT)
                 : dayjs(termindato).subtract(3, 'week').subtract(1, 'day').format(ISO_DATE_FORMAT);
 
-              const navn = `${tilretteleggingStateName}.${tilretteleggingIndex}`;
               return (
                 <TilretteleggingPeriodeTabellRad
-                  key={navn}
-                  navn={navn}
+                  key={`${tilretteleggingStateName}.${tilretteleggingIndex}`}
+                  navn={`${tilretteleggingStateName}.${tilretteleggingIndex}`}
                   tilrettelegging={rad}
                   readOnly={readOnly}
                   index={arbeidsforholdIndex + tilretteleggingIndex}
-                  openRad={rad.fom === undefined}
+                  openRad={rad.fom === ''}
```

Comment:
Denne har jeg vært borti i min WIP branch også. fom er jo her alltid definert, så er det riktig å sjekke mot tom streng? Så ut i backend som det isåfall er TIDENES_START eller no som er default. Vil det si at den alltid egentlig er åpen?